components/infobox/wikis/rocketleague/infobox_person_player_custom.lua

```diff
@@ -52,29 +52,36 @@ function CustomPlayer.run(frame)
 	return player:createInfobox(frame)
 end
 
+function CustomInjector:_parseActive(yearsActive, getYearsActive, getYearsActiveArg)
```

Comment:
```suggestion
function CustomInjector:_parseActive(manualInput, autoFunction, autoFunctionParam)
```
Not the perfect names, but I think they are more descriptive of what they actually do